airbyte-integrations/connectors/source-confluence/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_confluence ./source_confluence
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.version=0.1.1
```

Comment:
Why are you bumping the version? I can't see any change in this PR on the confluence connector code.

Reply:
Yeah the spec is not in cache so I thought the ideal way is to bump it and let it go through the process rather than doing it manually

Reply:
Sorry I don't get this: If you add the source connector with 0.1.0 to `source_definitions.yaml` and rerun ` ./gradlew airbyte-config:init:processResources` its not updating the `source_specs.yaml`? 
I think that version bumps should always be linked to a PR that modifies the code of the connector, which is not the case here: using 0.1.0 and 0.1.1 will be the same image.